Final-Project-Proposal-Group-22

jcscheufele opened the pull request.

Commit:
Checkpoint

Commit:
🎗🛎 Checkpoint
./index.html:8/101
./styles.css:8/77

Commit:
💅🚲 Checkpoint
./index.html:8/836
./styles.css:8/179

Commit:
💖🚒 Checkpoint
./index.html:96831/423
./styles.css:96831/354

Commit:
☔️👀 Checkpoint
./styles.css:96831/285
./index.html:96831/177
./script.js:96831/291
./README.md:96831/87

Commit:
🚶🍱 Checkpoint
./README.md:96831/3689

Commit:
📇🏂 Checkpoint
./index.html:96831/1
./README.md:96831/1

Commit:
🔔📕 Checkpoint

Commit:
🚌🏩 Checkpoint
./index.html:96831/4

Commit:
🍘👱 Checkpoint
./.glitch-assets:913293/86

Commit:
📻👋 Checkpoint
./.glitch-assets:963655/43

Commit:
📷🚇 Checkpoint
./README.md:334152/29

Commit:
🛬🍺 Checkpoint
./script.js:728660/1

Commit:
🌮📖 Checkpoint
./script.js:617503/320

Commit:
🍉🏏 Checkpoint
./style.css:867163/33

Commit:
Import of glitchdotcom/hello-webpage

Commit:
🦄🏆 Checkpoint
./style.css:617503/77
./index.html:617503/122

Commit:
🐠💃 Checkpoint
./style.css:617503/1560
./illustration.svg:617503/21274
./index.html:617503/2204

Commit:
🛩👢 Checkpoint
./style.css:617503/2419
./index.html:617503/291

Commit:
🏋😼 Checkpoint
./style.css:617503/2126
./.glitch-assets:617503/372
./index.html:617503/2766

Commit:
💁🍁 Checkpoint
./index.html:617503/7586
./style.css:617503/1840
./.glitch-assets:617503/1439

Commit:
🚎🏇 Checkpoint
./style.css:617503/606
./index.html:617503/120

Commit:
👚🚍 Checkpoint
./style.css:617503/665

Commit:
📮🎮 Checkpoint
./style.css:617503/4720
./index.html:617503/15

Commit:
Rewound to commit 34487b826755ba40659695a090bcca7c3e22d912 + 0 edits

Commit:
📼🎇 Checkpoint
./style.css:617503/127

Commit:
🕹🐈 Checkpoint
./style.css:617503/113

Commit:
👛💄 Checkpoint
./index.html:617503/32

Commit:
👍👭 Checkpoint
./style.css:617503/33
./index.html:617503/235

Commit:
🏙☎️ Checkpoint
./style.css:617503/11313
./index.html:617503/3007

Commit:
🍰🗻 Checkpoint
./index.html:617503/291
./style.css:617503/772

Commit:
🏔😺 Checkpoint
./index.html:1062301/1596

Commit:
🙏⏳ Checkpoint
./index.html:1062301/22

Commit:
🐾🐜 Checkpoint
./.gitignore:1062301/4
./index.html:1062301/80

Commit:
🚓🛀 Checkpoint
./README.md:1062301/1497

Commit:
🐜👮 Checkpoint
./README.md:1062301/569

Commit:
👚🌸 Checkpoint
./README.md:1062301/194

Commit:
💧🐊 Checkpoint
./.gitignore:617503/4

Commit:
🚿🍬 Checkpoint
./.glitch-assets:1062301/395
./index.html:1062301/593
./script.js:1062301/160

Commit:
👱👡 Checkpoint
./LICENSE:1062301/1465

Commit:
✂️🏑 Checkpoint
./LICENSE:1062301/119

Commit:
🤖👔 Checkpoint
./LICENSE:1062301/30619

Commit:
Exporting to glitchdotcom/glitch-hello-website
./LICENSE:1062301/2
./README.md:1062301/14

Commit:
👄🏎 Checkpoint
./LICENSE:1062301/1

Commit:
🕍🏩 Checkpoint
./LICENSE:1062301/12

Commit:
🍍🖐 Checkpoint
./index.html:617503/14

Commit:
⏲🍡 Checkpoint
./style.css:1062301/756
./.glitch-assets:1062301/172

Commit:
☔️🍏 Checkpoint
./index.html:1062301/195
./.glitch-assets:1062301/43

Commit:
🍇💳 Checkpoint
./index.html:1062301/6176
./style.css:1062301/5109

Commit:
🐔🍛 Checkpoint
./index.html:1062301/1049

Commit:
🕶💘 Checkpoint
./index.html:1062301/14

Commit:
🔋🏚 Checkpoint
./script.js:1062301/12
./style.css:1062301/2

Commit:
🎣🏂 Checkpoint
./README.md:1062301/7
./style.css:1062301/6407
./script.js:1062301/15

Commit:
🏥✌️ Checkpoint
./style.css:1062301/12

Commit:
🐚🍥 Checkpoint
./style.css:1062301/50

Commit:
🍕✈️ Checkpoint
./style.css:1062301/2104

Commit:
🏌❄️ Checkpoint
./style.css:14029292/399

Commit:
Import of glitchdotcom/glitch-hello-website

Commit:
Exporting to glitchdotcom/glitch-hello-website
./script.js:31317407/1858

Commit:
Import of glitchdotcom/glitch-hello-website

Commit:
Exporting to glitchdotcom/glitch-hello-website
./README.md:31317407/155

jcscheufele committed:
Create proposal.md

Commit:
🌨🏎 Checkpoint
./script.js:35500699/28:35541864/12
./server.js:35500699/42
./package.json:35500699/1207

Commit:
🗾🚂 Checkpoint
./server.js:35500699/183
./package.json:35500699/293:35533189/96

Commit:
👝🌥 Checkpoint
./server.js:35500699/319:35533189/68

charlieroberts commented:
Cool! This idea has a lot of UI elements to it that will require attention. For example, after the first player logs in, they need a notification to let them know they're waiting for the second player. Then there probably needs to be some type of countdown for the game to actually start, then you need a UI to restart the game and on and on. It's a lot! I think it's doable but it will require (at least) one person to really pay attention to the UI details and coordinate with the backend communication services. 

I would suggest coming up with a minimum viable project and then defining successive iterations that build up to your final idea... otherwise there's a danger of ending up with a bunch of individual pieces that don't quite come together at the end.

Commit:
🚉📡 Checkpoint
./views/login.html:35541864/538

Commit:
🐉🔑 Checkpoint
./public/js/loginscript.js:35541864/1656
./server.js:35541864/698

Commit:
💂🌞 Checkpoint
./server.js:35541864/3583

Commit:
⚽️🏀 Checkpoint

Commit:
🔑💏 Checkpoint
./public/js/blah.js:35500699/4671
./index.html:35500699/76
./views/game.html:35500699/547
./server.js:35500699/54
./public/js/loginscript.js:35500699/6
./views/login.html:35541864/19

Commit:
🍥📺 Checkpoint
./views/login.html:35541864/214
./server.js:35500699/121
./public/js/blah.js:35500699/3
./public/js/loginscript.js:35541864/404
./views/highscore.html:35541864/1027
./views/game.html:35500699/146

Commit:
🚓🐿 Checkpoint
./views/game.html:35500699/796
./views/login.html:35541864/217
./public/highscore1.html:35541864/163

Commit:
🌔🕋 Checkpoint
./views/login.html:35541864/22
./views/game.html:35500699/301:35533189/10
./server.js:35541864/290
./views/highscore.html:35541864/169

Commit:
👗🍃 Checkpoint
./views/game.html:35500699/3040
./views/login.html:35541864/67
./views/highscore.html:35541864/4
./server.js:35500699/18:35541864/48
./views/snake.html:35500699/11
./public/js/snake.js:35541864/12
./public/js/loginscript.js:35500699/18

Commit:
👖🎺 Checkpoint
./views/covids.html:35500699/300
./server.js:35533189/600
./README.md:35500699/2248

Commit:
🍳🐘 Checkpoint
./public/js/covids.js:35500699/539

Commit:
🚝💋 Checkpoint
./public/js/covids.js:35500699/1330

Commit:
🌞👐 Checkpoint
./public/js/covids.js:35500699/267

Commit:
🏙🔮 Checkpoint
./public/css/blah.css:35379689/10

Commit:
🍉🎸 Checkpoint
./views/covids.html:35379689/829
./public/css/blah.css:35379689/193
./views/login.html:35379689/369

Commit:
🛀🍝 Checkpoint
./views/login.html:35379689/483

Commit:
🖖🕵 Checkpoint
./views/login.html:35379689/654

Commit:
🌝🍏 Checkpoint
./.glitch-assets:35379689/377

Commit:
🌃🌾 Checkpoint
./views/login.html:35379689/68
./public/js/loginscript.js:35379689/240
./.glitch-assets:35379689/397

Commit:
💰🎢 Checkpoint
./public/js/loginscript.js:35379689/1777
./.glitch-assets:35379689/440
./public/css/blah.css:35379689/41
./views/login.html:35379689/419

Commit:
🚍🎺 Checkpoint
./views/login.html:35379689/298
./public/js/loginscript.js:35379689/131

Commit:
💽😀 Checkpoint
./views/login.html:35379689/1923
./public/js/loginscript.js:35379689/657

Commit:
🏘🚄 Checkpoint
./views/login.html:35379689/1514
./public/js/loginscript.js:35379689/15

Commit:
🍡🎍 Checkpoint
./views/login.html:35379689/383
./public/js/loginscript.js:35379689/18

Commit:
🐓🐗 Checkpoint
./views/login.html:35379689/300

Commit:
😬🎭 Checkpoint
./views/covids.html:35500699/1

Commit:
🖐🍕 Checkpoint
./public/js/covids.js:35500699/20

Commit:
👓✍️ Checkpoint
./public/js/covids.js:35500699/3110
./views/snake.html:35500699/11

Commit:
⛲️👲 Checkpoint
./public/js/covids.js:35500699/1233

Commit:
⛸💸 Checkpoint
./public/js/covids.js:35500699/1223
./.glitch-assets:35500699/1549

Commit:
🙆😬 Checkpoint
./public/js/covids.js:35500699/304

Commit:
🌑🍭 Checkpoint
./public/js/covids.js:35500699/759

Commit:
👼🌮 Checkpoint
./public/js/covids.js:35500699/690

Commit:
🔨🍪 Checkpoint
./public/js/covids.js:35500699/1787

Commit:
☔️👄 Checkpoint
./public/js/covids.js:35500699/647

Commit:
🚦🌵 Checkpoint
./public/js/covids.js:35500699/1229

Commit:
🍈👆 Checkpoint
./public/js/covids.js:35500699/1397

Commit:
😇🍜 Checkpoint
./public/js/covids.js:35500699/137

Commit:
🎌🐗 Checkpoint
./public/js/covids.js:35500699/2813

Commit:
🔔🕋 Checkpoint
./public/js/covids.js:35500699/299

Commit:
🏬🎆 Checkpoint
./public/js/covids.js:35500699/1373

Commit:
👵🐴 Checkpoint
./public/js/covids.js:35500699/23

Commit:
📖🚲 Checkpoint
./public/js/covids.js:35500699/248

Commit:
🛰🤑 Checkpoint
./public/js/covids.js:35500699/68

Commit:
🌇⛵️ Checkpoint
./public/js/covids.js:35500699/675
./server.js:35533189/7891

Commit:
🍼🏖 Checkpoint
./public/js/covids.js:35500699/37
./server.js:35533189/852

Commit:
🍲🚄 Checkpoint
./server.js:35533189/1773
./public/js/covids.js:35500699/211

Commit:
💈🕶 Checkpoint
./server.js:35533189/365
./public/js/covids.js:35500699/702

Commit:
🐕✊ Checkpoint
./public/js/covids.js:35500699/384
./server.js:35533189/703

Commit:
📻🌨 Checkpoint
./public/js/covids.js:35500699/673
./server.js:35533189/745

Commit:
🐟🍀 Checkpoint
./public/js/covids.js:35500699/209
./server.js:35533189/666

Commit:
🚲📉 Checkpoint
./public/js/covids.js:35500699/584
./server.js:35533189/110

Commit:
🌌👓 Checkpoint
./public/js/covids.js:35500699/68:35533189/471

Commit:
🌦💍 Checkpoint
./server.js:35533189/148
./public/js/covids.js:35500699/1021:35533189/1797

Commit:
🕰🚁 Checkpoint
./public/js/covids.js:35500699/705
./server.js:35533189/69

Commit:
🎬🐗 Checkpoint
./public/js/covids.js:35500699/385
./server.js:35533189/41

Commit:
💝🐙 Checkpoint
./public/js/covids.js:35500699/287:35533189/3469

Commit:
🍵🌹 Checkpoint
./public/js/covids.js:35500699/870:35533189/190
./server.js:35533189/4

Commit:
🎪😬 Checkpoint
./public/js/covids.js:35500699/389:35533189/123
./server.js:35533189/1956

Commit:
🎩🍰 Checkpoint
./public/js/covids.js:35500699/390
./views/snake.html:35533189/162

Commit:
🗼🐯 Checkpoint
./public/js/covids.js:35500699/152:35533189/1386
./views/snake.html:35533189/154

Commit:
☄️🍘 Checkpoint
./public/js/covids.js:35500699/533
./server.js:35533189/587

Commit:
💓🍩 Checkpoint
./public/js/covids.js:35500699/29

Commit:
😇🎨 Checkpoint

Commit:
🧀🚝 Checkpoint
./public/js/covids.js:35533189/136

Commit:
⭐️🐯 Checkpoint
./public/js/covids.js:35533189/1110
./server.js:35533189/2666

Commit:
💰⛴ Checkpoint
./public/js/covids.js:35533189/890
./server.js:35533189/388

Commit:
👋🐃 Checkpoint
./server.js:35533189/255
./public/js/covids.js:35533189/1733
./views/snake.html:35533189/90

Commit:
🌒🕋 Checkpoint
./public/js/covids.js:35533189/631
./server.js:35533189/4

Commit:
🚒🌹 Checkpoint
./views/snake.html:35533189/49
./public/css/style.css:35533189/408

Commit:
💄🌚 Checkpoint
./public/js/covids.js:35533189/152

Commit:
📖🌻 Checkpoint
./public/js/covids.js:35533189/552

Commit:
🏎🍔 Checkpoint
./public/js/covids.js:35533189/518

Commit:
🍎📠 Checkpoint
./public/js/covids.js:35533189/6145
./server.js:35533189/905

Commit:
⛳️⛴ Checkpoint
./public/js/covids.js:35533189/445
./server.js:35533189/74

Commit:
👧🍡 Checkpoint
./public/js/covids.js:35533189/1558
./server.js:35533189/815

Commit:
🏵🏙 Checkpoint
./public/js/covids.js:35533189/1473
./server.js:35533189/2100

Commit:
👚📇 Checkpoint
./public/js/covids.js:35500699/445

Commit:
🍔🍟 Checkpoint
./public/js/covids.js:35500699/40

Commit:
🌏💼 Checkpoint
./public/js/covids.js:35500699/18

Commit:
😜🎠 Checkpoint
./public/js/loginscript.js:35533189/297:35541864/28
./public/js/covids.js:35500699/101
./server.js:35541864/22
./views/snake.html:35533189/47
./views/covids.html:35533189/894
./views/login.html:35533189/139

Commit:
🐢👨‍❤️‍💋‍👨 Checkpoint
./views/login.html:35533189/329
./views/highscore.html:35379689/32:35500699/1:35533189/18
./views/covids.html:35379689/47:35500699/473
./server.js:35533189/58
./public/js/covids.js:35500699/69

Commit:
🚇💁 Checkpoint
./views/highscore.html:35379689/459
./views/covids.html:35500699/407
./public/js/covids.js:35500699/128:35533189/2809
./views/login.html:35379689/975
./server.js:35533189/562:35541864/76

Commit:
🌖💂 Checkpoint
./public/js/covids.js:35533189/112
./server.js:35541864/23971
./views/login.html:35379689/4410

Commit:
👽📫 Checkpoint
./public/js/covids.js:35533189/549
./.glitch-assets:35379689/806
./views/highscore.html:35379689/296
./server.js:35541864/388

Commit:
🐺👮 Checkpoint
./public/js/covids.js:35533189/337
./views/login.html:35500699/322
./server.js:35533189/508:35541864/324
./views/highscore.html:35379689/88:35500699/178
./views/covids.html:35379689/103:35500699/173
./public/css/style.css:35379689/100

Commit:
⛄️🚓 Checkpoint
./public/js/covids.js:35500699/1413:35533189/370
./public/css/style.css:35379689/307
./views/covids.html:35379689/1025:35500699/79

Commit:
🐕🚴 Checkpoint
./views/covids.html:35500699/487
./public/js/covids.js:35500699/32:35533189/346
./server.js:35541864/25612
./views/highscore.html:35379689/996:35500699/324
./views/login.html:35500699/324

Commit:
📈🏁 Checkpoint
./public/js/covids.js:35500699/2987:35533189/67:35541864/31452
./views/login.html:35379689/96:35500699/121
./views/highscore.html:35379689/41
./views/covids.html:35379689/77

Commit:
🙆👝 Checkpoint
./views/covids.html:35379689/53
./server.js:35541864/1
./public/js/scores.js:35541864/431
./public/js/covids.js:35500699/878
./views/highscore.html:35379689/94:35541864/69

Commit:
🍜🦃 Checkpoint
./public/js/covids.js:35500699/376
./views/highscore.html:35533189/22:35541864/38

Commit:
☕️🎎 Checkpoint
./views/highscore.html:35533189/65
./public/js/covids.js:35500699/74
./server.js:35541864/103
./public/js/scores.js:35541864/152

Commit:
🐇🍣 Checkpoint
./public/js/covids.js:35500699/812:35533189/13
./server.js:35541864/54
./public/js/scores.js:35541864/28

Commit:
🍧🚓 Checkpoint
./public/js/covids.js:35500699/124

Commit:
🌥🌝 Checkpoint
./views/highscore.html:35379689/697
./public/js/covids.js:35500699/8:35533189/291

Commit:
👾🏨 Checkpoint
./views/highscore.html:35379689/517
./public/js/covids.js:35500699/283:35533189/65
./public/js/scores.js:35379689/32
./views/covids.html:35533189/294

Commit:
🚩🚥 Checkpoint
./views/highscore.html:35379689/251
./public/js/scores.js:35379689/10:35500699/214
./public/js/covids.js:35500699/129

Commit:
✈️👛 Checkpoint
./public/js/scores.js:35379689/6:35500699/599
./views/highscore.html:35500699/395

Commit:
👔🎎 Checkpoint
./public/js/scores.js:35500699/683
./views/highscore.html:35379689/114:35500699/8:35533189/131
./public/js/covids.js:35500699/267

Commit:
👻🛢 Checkpoint
./public/js/covids.js:35500699/274:35533189/644
./server.js:35533189/40

Commit:
🕍📫 Checkpoint
./views/covids.html:35533189/24

Commit:
done

jcscheufele committed:
Update README.md

Commit:
hello

jcscheufele committed:
Update README.md
